packages/registry/src/registry/registry.ts

```diff
@@ -3,14 +3,16 @@ import {ComponentType} from 'react';
 import {ComponentMetadata} from './component-metadata';
 import Metadata from './metadata';
 import {getCompName} from '../utils/get-comp-name';
+import {getCompId} from '../utils/get-comp-id';
 import {isValidComponentName} from '../utils';
 
 const Registry: IRegistry = {
   metadata: new Metadata(),
   getComponentMetadata <Props, State>(comp: ComponentType<Props>): ComponentMetadata<Props, State> {
     const newCompName = getCompName(comp);
+    const newCompId = getCompId(comp);
 
-    if (!newCompName) {
+    if (!newCompName || !newCompId) {
```

Comment:
We don't need to check this here. This is for testing the presence of name of displayName, not ID. Maybe add another test for ID with a different error message.

Reply:
ok